app/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "daydream-scope-desktop",
+  "productName": "Daydream Scope",
+  "version": "0.1.0-alpha.7",
+  "description": "Daydream Scope - Real-time AI video generation and streaming",
```

```suggestion
  "description": "A tool for running and customizing real-time, interactive generative AI pipelines and models",
```

to match description used in pyproject.toml.